apertium/transfer.cc

```diff
@@ -304,13 +306,48 @@ Transfer::evalString(xmlNode *element)
       case ti_clip_sl:
         if(checkIndex(element, ti.getPos(), lword))
         {
+          if(ti.getContent().compare("lem") == 0 || ti.getContent().compare("lemh") == 0) //only get secondary tags if we're getting the lemma from the word
```

Comment:
```suggestion
          if(gettingLemmaFrimWorf(ti)) 
```

Maybe this could be written as